gyp ERR! configure error 
gyp ERR! stack Error: Command failed: Traceback (most recent call last):
gyp ERR! stack   File "/usr/lib/python2.7/site.py", line 562, in <module>
gyp ERR! stack     main()
gyp ERR! stack   File "/usr/lib/python2.7/site.py", line 544, in main
gyp ERR! stack     known_paths = addusersitepackages(known_paths)
gyp ERR! stack   File "/usr/lib/python2.7/site.py", line 271, in addusersitepackages
gyp ERR! stack     user_site = getusersitepackages()
gyp ERR! stack   File "/usr/lib/python2.7/site.py", line 246, in getusersitepackages
gyp ERR! stack     user_base = getuserbase() # this will also set USER_BASE
gyp ERR! stack   File "/usr/lib/python2.7/site.py", line 236, in getuserbase
gyp ERR! stack     USER_BASE = get_config_var('userbase')
gyp ERR! stack   File "/usr/lib/python2.7/sysconfig.py", line 577, in get_config_var
gyp ERR! stack     return get_config_vars().get(name)
gyp ERR! stack   File "/usr/lib/python2.7/sysconfig.py", line 481, in get_config_vars
gyp ERR! stack     _CONFIG_VARS['userbase'] = _getuserbase()
gyp ERR! stack   File "/usr/lib/python2.7/sysconfig.py", line 206, in _getuserbase
gyp ERR! stack     return env_base if env_base else joinuser("~", ".local")
gyp ERR! stack   File "/usr/lib/python2.7/sysconfig.py", line 192, in joinuser
gyp ERR! stack     return os.path.expanduser(os.path.join(*args))
gyp ERR! stack   File "/usr/lib/python2.7/posixpath.py", line 260, in expanduser
gyp ERR! stack     userhome = pwd.getpwuid(os.getuid()).pw_dir
gyp ERR! stack KeyError: 'getpwuid(): uid not found: 120117573'
gyp ERR! stack 
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:637:15)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at maybeClose (child_process.js:735:16)
gyp ERR! stack     at Socket.<anonymous> (child_process.js:948:11)
gyp ERR! stack     at Socket.EventEmitter.emit (events.js:95:17)
gyp ERR! stack     at Pipe.close (net.js:466:12)
gyp ERR! System Linux 3.2.0-36-generic
gyp ERR! command "node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /var/www/juka/node_modules/socket.io/node_modules/socket.io-client/node_modules/ws
gyp ERR! node -v v0.10.22
gyp ERR! node-gyp -v v0.11.0
gyp ERR! not ok 
